packages/manager/modules/anthos/src/anthos.html

```diff
@@ -0,0 +1,66 @@
+<oui-header data-heading="{{ ::'anthos_tenants_title' | translate }}">
+    <oui-guide-menu data-text="{{ ::$ctrl.guides.title }}">
+        <oui-guide-menu-item
+            data-ng-repeat="guideItem in $ctrl.guides.list track by guideItem.url"
+            data-href="{{ ::guideItem.url }}"
+            data-external="guideItem.external"
+            ><span data-ng-bind="::guideItem.name"></span>
+        </oui-guide-menu-item>
+        <oui-guide-menu-divider></oui-guide-menu-divider>
+        <oui-guide-menu-item
+            data-href="{{ ::$ctrl.guides.footer.url }}"
+            data-external="$ctrl.guides.footer.external"
```

Comment:
external seems to be always true here. Could be just `data-external`

Reply:
Removed, we don't need it.